helpers/deploymentConfig.ts

```diff
@@ -126,6 +127,13 @@ export const preconfiguredAddresses = {
     LzEndpoint: "0x012f6eaE2A0Bf5916f48b5F37C62Bcfb7C1ffdA1",
     LzVirtualChainId: "10333",
   },
+  unichainmainnet: {
+    NormalTimelock: UNICHAIN_MAINNET_MULTISIG,
+    FastTrackTimelock: UNICHAIN_MAINNET_MULTISIG,
+    CriticalTimelock: UNICHAIN_MAINNET_MULTISIG,
+    LzEndpoint: "0xb6319cC6c8c27A8F5dAF0dD3DF91EA35C4720dd7",
+    LzVirtualChainId: "320",
+  },
 };
 
 export const xvsBridgeMethodsSrc = [
```

Comment:
We will have to add the configuration of the XVS Bridge on mainnet:
* connected with every mainnet network
* limits set to zero